providers/aws/eks/eks.go

```diff
@@ -266,4 +266,30 @@ func DeleteExpiredEKSClusters(svc eks.EKS, ec2Session ec2.EC2, elbSession elbv2.
 		}
 
 	}
+}
+
+func TagClustersResources(svc eks.EKS, ec2Session ec2.EC2, rdsSession rds.RDS, tagName string) error {
```

Comment:
Please add an option disabled by default to tag Key associated to cluster. Because you can technically have 1 key for multiple clusters and if you delete it because you delete one cluster, a lot of things may not work like autoscalers.
As this is very Qovery specific, please add an option on the CLI